packages/wdio-cucumber-framework/src/utils.ts

```diff
@@ -108,7 +108,7 @@ export function buildStepPayload(
 ) {
     return {
         uid: step.id,
-        title: isStepTypeHook(step) ? `hook-${step.hookId}` : step.text,
+        title: isStepTypeHook(step) ? 'Hook' : step.text,
```

Comment:
Hi @rickschubert 

@christian-bromann and I took a look at it and we need to change more code to prevent Allure and other reporters for Cucumber to work with the never version of CucumberJS (we're also missing tags, keywords, datatables and so on).
I'll dive into this next week, but for now we think the fix can be made a bit simpler with the following

Change 

`title: isStepTypeHook(step) ? 'Hook' : step.text,`

to

`title: step.text || 'Hook',`

So it should work for your situation. That means you can remove the rest of your code. Can you update that so we can approve it and release it?

Reply:
Okay, I can amend this.